flocker/control/httpapi.py

```diff
@@ -544,7 +576,8 @@ def create_container_configuration(
             environment=environment,
             restart_policy=policy,
             cpu_shares=cpu_shares,
-            memory_limit=memory_limit
+            memory_limit=memory_limit,
+            links=frozenset(application_links)
```

Comment:
Don't need the `frozenset`; pyrsistent is just going to convert whatever you pass in to a `PSet`.